tests/utils/test_file_lock.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
I don't think this test is necessary for PaddleNLP since it's about the basic property of FileLock

Reply:
Almost all of python files contain this `Copyright` description.  You can modify the copyright rule in the new CI PR, and discuss with others. How do you think about it ?

Reply:
I don't mean the license. I meant that this entire test is not necessary since it is basic usage of FileLock.